pageserver/src/tenant/timeline.rs

```diff
@@ -1026,6 +1055,8 @@ impl Timeline {
         use super::layer_map::Replacement;
 
         if local_layer.is_remote_layer() {
+            // TODO: consider returning an err here instead of false, which is the same out the
```

Comment:
Should it be addressed before the merge?

Reply:
No, we have https://github.com/neondatabase/neon/issues/3851  to track general improvements to return signature. Will add a link to the TODO